solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -528,6 +561,57 @@ contract RandomBeacon is IRandomBeacon, Ownable {
         );
     }
 
+    /// @notice Updates the DKG result submission gas.
+    /// @dev Can be called only by the contract owner, which should be the
+    ///      wallet registry governance contract. The caller is responsible for
```

Comment:
Not the wallet registry governance ser.

```suggestion
    ///      random beacon governance contract. The caller is responsible for
```

Reply:
[dd9ad55](https://github.com/keep-network/keep-core/pull/2860/commits/dd9ad55739999e3d5bdc88bdc68b84b1c20367f9)